__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const db = require("../db/connection.js");
+const app = require("../app");
+const request = require("supertest");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index.js");
+const seed = require("../db/seeds/seed.js");
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+
+afterAll(() => db.end());
+
+describe("GET: /api/topics", () => {
+  it("should respond with a json object containing an array of all the topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const topics = body;
+        expect(topics).toBeInstanceOf(Array);
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            description: expect.any(String),
+            slug: expect.any(String),
+          });
+        });
+      });
+  });
```

Comment:
Great test, though we're missing the one sad path test we can get at this point, i.e. a request to a bad path like `/api/toopics` or `/api/katherine` for instance